docs/dev/onnx_operators.rst

```diff
@@ -695,7 +695,13 @@ Operator Support Matrix
 +--------------------------+-----------+-----------------+------------------------------+
 | STFT                     | ❌        |                 |                              |
 +--------------------------+-----------+-----------------+------------------------------+
-| Scan                     | 👷        | 👷              |                              |
+| Scan                     | ✅        | UINT8, UINT16,  | ``identity``,                |
+|                          |           | UINT32, UINT64, | ``sequence``                 |
```

Comment:
Why identity and sequence are mentioned here ?

Reply:
They're mentioned in the Loop op entry as well. Since scan relies on Loop, I thought I'd carry over the info.

Reply:
Okay. I do not know what that means though. @attila-dusnoki-htec  can you comment this limitation is about ?